crates/diesel_models/src/business_profile.rs

```diff
@@ -475,12 +475,12 @@ impl BusinessProfileUpdateInternal {
     }
 }
 
-// This is being used only in the `BusinessProfileInterface` implementation for `MockDb`.
-// This can be removed once the `BusinessProfileInterface` trait has been updated to use the domain
+// This is being used only in the `ProfileInterface` implementation for `MockDb`.
+// This can be removed once the `ProfileInterface` trait has been updated to use the domain
 // model instead.
 #[cfg(feature = "v2")]
-impl From<BusinessProfileNew> for BusinessProfile {
-    fn from(new: BusinessProfileNew) -> Self {
+impl From<ProfileNew> for Profile {
```

Comment:
Can we remove this impl since we're now using the domain model in the interface?